### ISSUE TITLE
TouchID and Exec is still sandboxed

### ISSUE THREAD

**Issue by ghost** [tihmstar/doubleH3lix#2]
Bytafont and Cydia Eraser is still crashing at launch and need to be fixed thru filza. Another this is that my touchid still doesnt work but now it doesnt work in nonjailed mode.

**Comment by ghost** [comment]
Oh ya btw in on ip6s+ 10.3.3 made the switch from g0blin RC2

**Comment by GeekBreak** [comment]
Cydia apps crashing isn't due to The Jailbrrak rather than APFS itself, launchscript and stashed binary doesn't work, and probabili this can't be fixed (see 32bit H3lix, since there isn't APFS on 32bit devices, those apps works)

About TouchID issues, probably seitchung eirhout a reagire from g0blin which have the TouchID bug, keeps the TouchID still broken

**Comment by ghost** [comment]
Ok so first of all. Thank you. Second, anyway to completely uninstall g0blin? Should i run Eraser?

**Comment by GeekBreak** [comment]
Yep, successfully used Eraser on iOS 10.3.1 (iPhone 6S) and I have no issues with doubleH3lix except the bad success rate due to v0rtex exploit itself

**Comment by Peeprince** [comment]
I can not unzip files in filza and i can not ssh into the phone.

**Comment by parazyt75** [comment]
cydia eraser fix 
http://www.redmondpie.com/enable-cydia-eraser-on-ios-10.3.3-jailbreak-and-fix-errors-heres-how/

**Comment by parazyt75** [comment]
Cydia Eraser fix
http://www.redmondpie.com/enable-cydia-eraser-on-ios-10.3.3-jailbreak-and-fix-errors-heres-how/

**Comment by nik312123** [comment]
This includes the BytaFont3 fix: https://www.reddit.com/r/jailbreak/comments/81j8eg/tutorial_how_to_make_cocoatop_bytafont_3/?st=JEJ5LKE2&sh=652e6e7d
